lib/SIL/SILPrinter.cpp

```diff
@@ -3026,6 +3057,72 @@ void SILDefaultWitnessTable::dump() const {
   print(llvm::errs());
 }
 
+// SWIFT_ENABLE_TENSORFLOW
+void SILDifferentiabilityWitness::print(
+    llvm::raw_ostream &OS, bool verbose) const {
+  OS << "// differentiability witness for "
+     << demangleSymbol(originalFunction->getName()) << "\n";
+  // sil_differentiability_witness @original-function-name : $original-sil-type
+  PrintOptions qualifiedSILTypeOptions = PrintOptions::printQualifiedSILType();
+  OS << "sil_differentiability_witness ";
+  printLinkage(OS, linkage, ForDefinition);
+  // [parameters 0 1 ...]
+  OS << "[parameters ";
+  interleave(getParameterIndices()->getIndices(),
+             [&](unsigned index) { OS << index; },
+             [&] { OS << " "; });
+  // [results 0 1 ...]
+  OS << "] [results ";
+  interleave(getResultIndices()->getIndices(),
+             [&](unsigned index) { OS << index; },
+             [&] { OS << " "; });
```

Comment:
```suggestion
             [&] { OS << ' '; });
```

Reply:
Done in https://github.com/apple/swift/pull/27487/commits/2073f863826fb305c1fc105a520d84de1e1463ec.